api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```
@@ -260,19 +261,21 @@ private static String generateRandomChars(String candidateChars, int length) {
     return sb.toString();
   }
 
-  private void setCdrVersionId(org.pmiops.workbench.db.model.Workspace dbWorkspace, String cdrVersionId) {
-    if (cdrVersionId != null) {
-      try {
-        CdrVersion cdrVersion = cdrVersionDao.findOne(Long.parseLong(cdrVersionId));
-        if (cdrVersion == null) {
-          throw new BadRequestException(
-              String.format("CDR version with ID %s not found", cdrVersionId));
-        }
-        dbWorkspace.setCdrVersion(cdrVersion);
-      } catch (NumberFormatException e) {
-        throw new BadRequestException(String.format(
-            "Invalid cdr version ID: %s", cdrVersionId));
+  private CdrVersion setCdrVersionId(org.pmiops.workbench.db.model.Workspace dbWorkspace, String cdrVersionId) {
+    if (Strings.isNullOrEmpty(cdrVersionId)) {
+      return null;
```

This should throw an exception, yes?